extensions/wikia/PortableInfobox/services/PortableInfoboxMobileRenderService.class.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+use Wikia\PortableInfobox\Helpers\PortableInfoboxRenderServiceHelper;
+
+class PortableInfoboxMobileRenderService extends PortableInfoboxRenderService {
+	const MEDIA_CONTEXT_INFOBOX_HERO_IMAGE = 'infobox-hero-image';
+	const MEDIA_CONTEXT_INFOBOX = 'infobox';
+
+	protected $imagesWidth = PortableInfoboxRenderServiceHelper::MOBILE_THUMBNAIL_WIDTH;
+
+	/**
+	 * renders infobox
+	 *
+	 * @param array $infoboxdata
+	 *
+	 * @param $theme
+	 * @param $layout
+	 * @param $accentColor
+	 * @param $accentColorText
+	 * @return string - infobox HTML
+	 */
+	public function renderInfobox( array $infoboxdata, $theme, $layout, $accentColor, $accentColorText ) {
```

Comment:
I think that 5 is close to edge but still it is not a pain. Introducing "options" array in my opinion would make it less readable when you'd like to use it somewhere as it does not describe what keys should be set